core/templates/dev/head/components/SearchBarDirective.js

```diff
@@ -121,6 +122,21 @@ oppia.directive('searchBar', [function() {
               $rootScope.$broadcast('hasChangedSearchQuery');
             }
           });
+
+          var searchUrl = searchService.searchQueryUrl(
```

Comment:
Having said that, is this the whole URL or just the query fragment? If the latter, a better naming convention would be something like

    var searchUrlQueryString = searchService.getSearchUrlQueryString(